.github/workflows/ci.yaml

```
@@ -17,6 +18,127 @@ jobs:
       - name: Run ah lint
         working-directory: ./charts/mlrun-ce
         run: ah lint
+
+  ci-runner:
```

Lets call this something more specific
```suggestion
  demos-notebook-tests:
```